frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -280,6 +343,17 @@ export default function CourseDescription({ structure }) {
                   </Tag>
                 );
               })}
+            <Title level={3} className="text cs-final-attr">
+              Capacity
```

Comment:
This should probably also include what term the capacity is being shown for since it's not immediately clear whether during something like Week 10 of a term if we're showing the capacity for the current term or the next.